draft-ietf-teep-protocol.md

```diff
@@ -440,8 +440,10 @@ token
   QueryRequest.
 
 selected-cipher-suite
-: The selected-cipher-suite parameter indicates the selected ciphersuite. Details
-  about the ciphersuite encoding can be found in {{ciphersuite}}.
+: The selected-cipher-suite parameter indicates the selected ciphersuite. If this
+  parameter is not present, it is to be treated as if the TEEP Agent accepts
+  any ciphersuites listed in the QueryRequest, and the TAM SHOULD select it.
```

Comment:
```suggestion
  any ciphersuites listed in the QueryRequest, and the TAM can select it.
```
There's no obvious alternative to the SHOULD, so change language.

(My preference would actually me to make the selected-cipher-suite mandatory instead of optional though, but I am ok either way.)